### PR TITLE
fixed issue #202 - removing symlinks

### PR DESCRIPTION
**Pull request by fzzr-**

### CODE REVIEW

**Comment by fzzr-** [comment]
@back2dos : How should I add tests to appveyor.yml?

**Comment by back2dos** [comment]
The appveyor calls haxe tests.hxml, which ultimately ends up here: https://github.com/HaxeFoundation/haxelib/blob/master/test/HaxelibTests.hx#L49

**Comment by fzzr-** [comment]
Hmmm... What you think about https://github.com/fzzr-/haxelib/commit/a10c235789a93ebebe85de5352d3ea0ab41d417b#diff-354f30a63fb0907d4ad57269548329e3R9 ? :(

**Comment by back2dos** [comment]
Uhm, this really isn't my strong suit, so I have to ask: why not just add it in the `HaxelibTests`? Am I missing something?

**Comment by fzzr-** [comment]
My mistake. I'll do it very soon.

**Comment by waneck** [comment]
I hate how our sys API is lacking a more sane way to detect this.

>

File: test/tests/TestRemoveSymlinks.hx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		super();
+	}
+
+
+	//--------------- initialize ----------------//
+
+	override public function setup():Void
+	{
+		Sys.command("neko", ["./bin/haxelib.n", "setup", Sys.getCwd() + "haxelib-repo"]);
+		Sys.command("neko", ["./bin/haxelib.n", "install", "nme-dev", "1.3.2", "--always"]);
```
Comment (andyli) [inline test/tests/TestRemoveSymlinks.hx]: Please do not depends on 3rd party libs in our test. They could take long time to download. 
Create only a minimal lib in `/test/libraries` and use that.

Comment (back2dos) [inline test/tests/TestRemoveSymlinks.hx]: Well, I think using something smaller than nme-dev should not be that much of an issue, e.g. mcli, or whetever else you find under 100KB.

**Comment by fzzr-** [comment]
__Ready for merge.__
@waneck : I agree and I have some ideas for std..

**Pull request merged**